Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -186,6 +188,7 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
       return null;
     }
 
+    const {subviewStyle = {} } = this.props;
```

Comment:
We're unnecessarily creating an empty object, though not major, could be easily avoided with a simple check when accessing a property